Jenkinsfile

```diff
@@ -525,6 +525,17 @@ void TestInVM(deviceMode, deploymentMode, distro, distroVersion, kubernetesVersi
                                          TEST_E2E_SKIP=\$(if [ \"${env.CHANGE_ID}\" ] && [ \"${env.CHANGE_ID}\" != null ]; then echo \\\\[Slow\\\\]; fi) \
                            ' \
            "
+    } catch (exc) {
+        echo "Handling exception, trying to get state and logs:"
+        sh "_work/${env.CLUSTER}/ssh.0 kubectl get pods --all-namespaces -o wide"
+        echo "====== KUBELET LOG MASTER ======"
+        sh "_work/${env.CLUSTER}/ssh.0 sudo journalctl -u kubelet"
+        echo "====== KUBELET LOG WORKER-1 ======"
```

Comment:
Please don't hard-code the number of nodes. You can iterate over `_work/${env.CLUSTER}/ssh.*` instead.

Reply:
removed hard coding, now runs using all ssh.* wrappers